brut.j.util/src/main/java/brut/util/OS.java

```diff
@@ -188,7 +188,7 @@ public void run() {
                 while ((line = reader.readLine()) != null) {
                     buffer.append(line).append('\n');
                 }
-            } catch (IOException ignored) {}
+            } catch (IOException ignored) {}            
```

Comment:
final comment - there is some extra spacing added to this line. Can it be removed?

Reply:
Removed ;)